_source/_posts/2021-08-11-okta-identity-engine-early-access-hackathon.md

```diff
@@ -0,0 +1,29 @@
+# Join the Okta Identity Engine Early Access Hackathon 2021 Challenge
```

Comment:
You need "front matter" for this to render properly. The title shouldn't be here, but rather in the front matter. If you look at other blog posts, you'll see some examples. Here's a snippet from one I recently updated:

```
---
layout: blog_post
title: "Spring Cloud Config for Shared Microservice Configuration"
author: joe-cavazos
by: contractor
communities: [java]
description: "This tutorial shows how to use Spring Cloud config to manage configuration properties for multiple Spring Boot applications from a single source."
tags: [java, spring-boot, spring-cloud, spring-cloud-config, oauth2, microservices]
image: blog/spring-cloud-config/spring-cloud-config.png
type: conversion
---
```

A few things that should change are type (awareness), by (to internal-contributor), and author.